data/clients.yaml

```diff
@@ -1,12 +1,16 @@
 featureGroups:
   - description: Data format and transport for reporting spans to Jaeger backend
     features:
-    - name: report_jaeger_thrift_udp
-      description: Report [`jaeger.thrift`](https://github.com/jaegertracing/jaeger-idl/blob/master/thrift/jaeger.thrift) over UDP
-    - name: report_jaeger_thrift_http
-      description: Report [`jaeger.thrift`](https://github.com/jaegertracing/jaeger-idl/blob/master/thrift/jaeger.thrift) over HTTP
-    - name: report_zipkin_thrift_http
-      description: Report Zipkin Thrift over HTTP
+    - name: report_jaeger_thrift_compact_udp
+      description: Report [`jaeger.thrift`](https://github.com/jaegertracing/jaeger-idl/blob/master/thrift/jaeger.thrift), [Compact](https://github.com/apache/thrift/blob/v0.13.0/doc/specs/thrift-compact-protocol.md) protocol, over UDP
+    - name: report_jaeger_thrift_binary_udp
+      description: Report [`jaeger.thrift`](https://github.com/jaegertracing/jaeger-idl/blob/master/thrift/jaeger.thrift), [Binary](https://github.com/apache/thrift/blob/v0.13.0/doc/specs/thrift-binary-protocol.md) protocol, over UDP
+    - name: report_jaeger_thrift_compact_http
```

Comment:
do we need this line? no client uses this format, and I am not even sure if the server supports it

Reply:
I actually confirmed in a previous comment that the server does not support Thrift Compact via HTTP, but you said that it's better if the matrix is explicit. If you now prefer to not have it listed at all, I can remove it, especially as it's also my preference ;-)

Reply:
let's remove what doesn't exist.

Reply:
Done!